src/compiler/llvm/llvmir.rs

```diff
@@ -484,6 +484,34 @@ impl<'ctx> IrGen<'ctx> {
         self.record_terminal(span, &mc);
     }
 
+    /// Returns the LLVM Intrinsic for FP to Signed Integer
+    fn get_fptosi(&self, src: &Type, target: &Type) -> FunctionValue<'ctx> {
+        let i_name = format!("llvm.fptosi.sat.{}.{}", src, target);
```

Comment:
We should preconstruct the list of allowed conversion intrinsics.

Reply:
This is because there is a strictly defined set that Bramble will support and those should be defined in one place.  This current design allows for any combination of types to add an intrinsic external declaration to Bramble code.

Reply:
No longer valid because we are not using intrinsics any more.